2024/json-ld-wg/index.html

```diff
@@ -225,10 +257,20 @@ <h3>
             <ul>
               <li>Application profile of JSON-LD to enable efficient streaming parsers.</li>
               <li>Test suites and implementation reports for the specifications.</li>
-              <li>JSON-LD 1.1 specified in other serializations like YAML or CBOR.</li>
               <li>Best practices for use and implementation of JSON-LD 1.1.</li>
             </ul>
         </section>
+
+        <section id="timeline">
+          <h3>Timeline</h3>
+            <ul>
+              <li>September 2022: First teleconference</li>
+              <li>Q2 2024: FPWD of CBOR-LD and YAML-LD</li>
+              <li>Q4 2024: Candidate Recommendation of CBOR-LD and YAML-LD</li>
+              <li>Q1 2025: Recommendation of CBOR-LD and YAML-LD</li>
```

Comment:
It's easy to read these line items as each being a single document covering both CBOR-LD and YAML-LD. I suggest a little restructuring.
```suggestion
              <li>Q2 2024
                <ul>
                  <li>FPWD of CBOR-LD</li>
                  <li>FPWD of YAML-LD</li>
                </ul>
              </li>
              <li>Q4 2024
                <ul>
                  <li>Candidate Recommendation of CBOR-LD</li>
                  <li>Candidate Recommendation of YAML-LD</li>
                </ul>
              </li>
              <li>Q1 2025
                <ul>
                  <li>Recommendation of CBOR-LD</li>
                  <li>Recommendation of YAML-LD</li>
                </ul>
              </li>
```